roles/ydbd_static/defaults/main.yaml

```
@@ -1,3 +1,10 @@
 ydb_allow_format_drives: false
 ydb_skip_data_loss_confirmation_prompt: false
 ydb_enforce_user_token_requirement: true
+# ydb_use_dynamic_config - define how to configure dynamic nodes - static or dynamic configs
+# Default value: false
+ydb_use_dynamic_config: false
```

Maybe it's time to introduce the concept of playbook feature flags. I don't think we'd want to keep this setting around long-term: it's more like to test this new approach, make it default at some point and then deprecate the current approach. So it could be like this:
```suggestion
ydb_ansible_feature_flags:
    use_dynamic_config: false
```

---

It's good idea fro single roles, but we might have problems in our structure
playbook1 (feature_flag: feature1, feature3) -> role1 (feature_flag: feature1)
playbook1 (feature_flag: feature1, feature3) -> role2 (feature_flag: feature1, feature2)
in this case role2 will lost feature2 default value

```
TASK [role1 : Output] *****************************************************************************************************************************************************************************************************
ok: [localhost] => {
    "feature_flags": {
        "feature1": true,
        "feature3": true
    }
}

TASK [role2 : Output] *****************************************************************************************************************************************************************************************************
ok: [localhost] => {
    "feature_flags": {
        "feature1": true,
        "feature3": true
    }
}
```

---

Let's make feature flags as separate PR. I think it's better to implement in next releases or version (v2)